testing/thunderbird-extension-enigmail/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=thunderbird-extension-enigmail
+pkgver=2.0.12
+pkgrel=1
+pkgdesc="OpenPGP message encryption and authentication for Thunderbird"
+url="https://www.enigmail.net/"
+arch="noarch"
+license="MPL GPL3"
+depends="thunderbird gnupg"
+makedepends="zip python2 perl"
```

Comment:
Switch to `python3` if at all possible.

Reply:
Sadly, the upstream requires python2 binary to be present... Hopefully they'll resolve it soon. If needed down the road, I'd consider making a patch for them.